protocols/CNRS/README.md

```diff
@@ -0,0 +1,34 @@
+# Spotting of Agar Plates
+
+### Author
+[Opentrons](http://www.opentrons.com/)
+
+### Partner
+CNRS
+
+## Categories
+* Basic Pipetting
+
+## Description
+This protocol allows the robot to place 2 drops of liquid 2 cm away from each other in petri dishes.
+
+Steps:
```

Comment:
Put in process. What sizes are the petri dishes? Do you need adaptors? What type of application is this? [Agar plate cultures]

Reply:
( I wasn't descriptive about this, but if you want to put in code comments that the 'from_center() command uses percentages and thus we created a percentage difference from the center)